data.sql

```diff
@@ -4,3 +4,10 @@ INSERT INTO animals (name, date_of_birth, escape_attemps, neutered,weight_kg ) V
 INSERT INTO animals (name, date_of_birth, escape_attemps, neutered,weight_kg ) VALUES ('Gabumon', '2018-11-15', 2, true, 8);
 INSERT INTO animals (name, date_of_birth, escape_attemps, neutered,weight_kg ) VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attemps, neutered,weight_kg ) VALUES ('Devimon', '2017-05-12', 5, true, 11);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Charmander', '2020-02-08', 0, false, 11);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', '2021-11-15', 2, true, 5.7);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Squirtle', '1993-04-02', 3, false, 12.13);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Angemon', '2005-06-12', 1, true, 45);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Boarmon', '2005-06-07', 7, true, 20.4);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Blossom', '1998-10-13', 3, true, 17);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Ditto', '2022-05-14', 4, true, 22);
```

Comment:
- [ ] Kindly ensure you enter the correct values, the weight is wrong in Charmander,  Plantmon,Squirtle and Angemon, this can lead to false results in your queries

Reply:
Thanks for the feedback, I will work on it and ask for a review again.